packages/material-ui-lab/src/Autocomplete/Autocomplete.test.js

```diff
@@ -1610,5 +1610,25 @@ describe('<Autocomplete />', () => {
       expect(handleChange.args[1][1]).to.equal(options[0]);
       expect(handleChange.args[1][2]).to.equal('mouse');
     });
+
+    it('should open list box even if the input contains an option value', () => {
```

Comment:
Perfect, we can keep this test only that it should fail. The "toggle" behavior should only work if the input is empty.

Regarding the issue at stake, it's about allowing to open the popup on the first click even if a value was already selected. It was working correctly in https://v4-8-3.material-ui.com/components/autocomplete/.

Reply:
Ok so he have two cases:
1) empty value: clicking on input is behavior is like "toggle"
2) with value: if the input not has focus it will open otherwise it remains open

Right?

Reply:
I think it's right